src/python/foglamp/data_purge/sqlalchemy_insert.py

```diff
@@ -0,0 +1,67 @@
+"""
+Date: June 21 2017 
```

Comment:
@Terris-Dianomic What do you mean by "except clauses"? Do we want no try/catch cases?